src/bigquery.ts

```diff
@@ -579,14 +582,16 @@ export class BigQuery extends Service {
    *     Please see {@link IntegerTypeCastOptions} for options descriptions.
    * @param {array} options.selectedFields List of fields to return.
    * If unspecified, all fields are returned.
-   * @param {array} options.parseJSON parse a 'JSON' field into a JSON object.
+   * @param {boolean} options.parseJSON parse a 'JSON' field into a JSON object.
+   * @param {boolean} options.skipWrapCustomTypes skip converting values into BigQuery custom types.
    * @returns Fields using their matching names from the table's schema.
    */
   static mergeSchemaWithRows_(
     schema: TableSchema | TableField | undefined,
     rows: TableRow[],
     options: {
       wrapIntegers: boolean | IntegerTypeCastOptions;
+      skipWrapCustomTypes?: boolean;
```

Comment:
nit - can this be the last option? I know order isn't necessarily preserved but I feel like we should have these options in the order in which we have added them

Reply:
additional comment on this line - Any reason why we can't call this wrapCustomTypes? I know it would flip the boolean logic later on, but it would be in line with the wrapIntegers naming scheme